src/main/java/Deadline.java

```diff
@@ -0,0 +1,22 @@
+public class Deadline extends Task {
+
+    protected String by;
```

Comment:
Hi! For this variable, I think it can be changed into private! Same applies to the other variables in other classes as well.